### PR TITLE
Added RN718N3 + MCF-LW12MET + EGK-LW22PLG

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/enginko/MCF-LW12MET/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      "type": "number"
+    },
+    "reactivePower": {
+      "title": "Reactive power",
+      "unit": "VAR",
+      "description": "Reactive power",
+      "type": "number"
+    },
+    "apparentPower": {
+      "title": "Apparent energy",
```
Comment (mariusgiger) [inline types/enginko/MCF-LW12MET/default.schema.json:38]: wrong title?

File: types/enginko/MCF-LW12MET/default.schema.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      "unit": "VA",
+      "description": "Apparent energy",
```
Comment (mariusgiger) [inline types/enginko/MCF-LW12MET/default.schema.json:40]: wrong description?

File: types/enginko/MCF-LW12MET/io.schema.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{
+  "$id": "https://akenza.io/enginko/EGK-LW12MET/IO.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "IO",
```
Comment (mariusgiger) [inline types/enginko/MCF-LW12MET/io.schema.json:4]: possibly find a more speaking name

File: types/enginko/MCF-LW12MET/uplink.spec.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    consume = utils.init(script);
+    utils
+      .loadSchema(`${__dirname}/default.schema.json`)
+      .then((parsedSchema) => {
+        defaultSchema = parsedSchema;
+        done();
+      });
+  });
+  /*
+  before((done) => {
```
Comment (mariusgiger) [inline types/enginko/MCF-LW12MET/uplink.spec.js:22]: remove or add

File: types/netvox/R718N3/lifecycle.schema.json [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+      "title": "Hardware version",
+      "description": "Hardware version",
+      "hideFromKpis": true
+    },
+    "dataCode": {
+      "title": "Data code",
+      "description": "Data code",
+      "hideFromKpis": true
+    },
+    "battery": {
```
Comment (mariusgiger) [inline types/netvox/R718N3/lifecycle.schema.json:22]: what is the unit of this?

File: types/netvox/R718N3/lifecycle.schema.json [hunk 6]
```diff
@@ -0,0 +1,28 @@
+{
+  "$id": "https://akenza.io/netvox/r718n3/lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Lifecycle",
+  "type": "object",
+  "properties": {
+    "softwareVersion": {
+      "title": "Software version",
```
Comment (mariusgiger) [inline types/netvox/R718N3/lifecycle.schema.json:8]: add data types

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, there are some comments that you should revisit.

File: types/enginko/MCF-LW12MET/io.schema.json (hunk 3 above)
Comment (J2thatsme) [reply types/enginko/MCF-LW12MET/io.schema.json:4]: I could do Input/Output
In other devices i also should change this then

**Review by sevyakenza — APPROVED** [review]

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**